Makefile

```diff
@@ -400,10 +412,9 @@ $(BUILD_OUTPUT_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-win32-$(TAR
 	| $(BUILD_OUTPUT_DIRECTORY)
 	./scripts/build/zip-file.sh -f $< -s $(TARGET_PLATFORM) -o $@
 
-$(BUILD_OUTPUT_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-win32-$(TARGET_ARCH).exe: \
-	$(BUILD_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-win32-$(TARGET_ARCH) \
-	| $(BUILD_OUTPUT_DIRECTORY) $(BUILD_TEMPORARY_DIRECTORY)
-	./scripts/build/electron-installer-nsis-win32.sh -n $(APPLICATION_NAME) -a $< -t $(BUILD_TEMPORARY_DIRECTORY) -o $@
+$(BUILD_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-win32-$(TARGET_ARCH).exe: \
+	| $(BUILD_DIRECTORY)
+	PATH=$(NODE_MODULES_BIN):$$PATH TARGET_ARCH=$(TARGET_ARCH) build --win nsis $(ELECTRON_BUILDER_OPTIONS)
```

Comment:
If run through an `npm run` script then `node_modules/.bin` is already in the path (details: `$ npm help run`) and all tools should resolve to binaries in there. I'd suggest adding 
```json
"electron-builder": "electron-builder"
```
to the scripts in the `package.json`, so you can run it as `ENV_VAR=value npm run electron-build -- build --win nsis` etc.

Alternatively, we could use [npx](https://github.com/zkat/npx), which is built to solve this particular problem of executing npm package binaries. This would also avoid cluttering the `package.json`.